k6/README.md

```diff
@@ -2,13 +2,13 @@
 
 ## Introduction
 
-This readme is for how to install and us k6 for manual load testing
+Grafana k6 is an open-source load testing tool that makes performance testing easy and productive for engineering teams. k6 is free, developer-centric, and extensible. In this project we are using k6 for manual load testing. The following readMe will give basic instructions on where, what and how to install and run.
 
 ## Installation
 
-The link for installation of k6 can be found [here](https://grafana.com/docs/k6/latest/get-started/installation/). Follow the instructions on the link for your installer of choice
+The link for installation of k6 can be found [Download and install k6](https://grafana.com/docs/k6/latest/get-started/installation/). Follow the instructions on the link for your installer of choice
```

Comment:
Remove the leading "The link for installation of k6 can be found" text, as it makes no sense with that still there.